src/impl/vamp/bindings/common.hh

```diff
@@ -150,17 +173,19 @@ namespace vamp::binding
             const ConfigurationArray &start,
             const ConfigurationArray &goal,
             const EnvironmentInput &environment,
-            const vamp::planning::RRTCSettings &settings) -> PlanningResult
+            const vamp::planning::RRTCSettings &settings,
+            typename RNG::Ptr &rng) -> PlanningResult
```

Comment:
Probably just pass the (shared) pointer by value, no? Unless it's worth using the reference to avoid incrementing/decrementing the use counter.